modules/EPMC.py

```diff
@@ -0,0 +1,179 @@
+
+    # Aggregating cooccurrence, get score apply filter:    
+    aggregated_df = (
+        filtered_cooccurrence_df
+
+        # Aggregating data by publication, target and disease:
+        .groupBy(['pmid', 'targetFromSourceId', 'diseaseFromSourceMappedId'])
+        .agg(
+            pf.first(pf.col('targetFromSource')).alias('targetFromSource'),
+            pf.first(pf.col('diseaseFromSource')).alias('diseaseFromSource'),
+            pf.collect_set(pf.col('tmp')).alias('literature'),
+            pf.collect_set(
+                pf.struct(  
+                    pf.col('text'),
+                    pf.col('start1').alias('tStart'),
+                    pf.col('end1').alias('tEnd'),
+                    pf.col('start2').alias('dStart'),
+                    pf.col('end2').alias('dEnd'),
+                    pf.col('section')
+                )
+            ).alias('textMiningSentences'),
+            pf.sum(pf.col('evidence_score')).alias('resourceScore')
+        )
+
+        # Summarizing all scores then filter for evidence string with at least score == 2
+        .filter(pf.col('resourceScore') > 1)
+    )
+
+    # Report number of evidence:
+    logging.debug(f'Number of evidence: {aggregated_df.count()}')
+
+    ##
+    ## Final formatting and saving data:
+    ##
+    (
+        aggregated_df
+
+        # Adding literal columns:
+        .withColumn('datasourceId', pf.lit('europepmc'))
+        .withColumn('datatypeId', pf.lit('literature'))
+
+        # Reorder columns:
+        .select(['datasourceId', 'datatypeId', 'targetFromSource', 'targetFromSourceId', 'resourceScore',
+                'diseaseFromSource','diseaseFromSourceMappedId','literature','textMiningSentences'])
+
+        # Save output:
+        .write.format('json').mode('overwrite').option('compression', 'gzip').save(outputFile)
+    )
+    
+    logging.info(f'EPMC disease target evidence saved.')
+
+
+def parse_args():
+
+    parser = argparse.ArgumentParser(description='This script generates target/disease evidence strings from ePMC cooccurrence files.')
+    parser.add_argument('--cooccurrenceFile', help='Partioned parquet file with the ePMC cooccurrences', type=str, required=True)
+    parser.add_argument('--outputFile', help='Resulting evidence file saved as compressed JSON.', type=str, required=True)
+    parser.add_argument('--logFile', help='Destination of the logs generated by this script.', type=str, required=False)
+    parser.add_argument('--local', help='Destination of the logs generated by this script.', action='store_true', required=False, default=False)
+    args = parser.parse_args()
+
+    # extract parameters:  
+    cooccurrenceFile = args.cooccurrenceFile
+    logFile = args.logFile
+    outputFile = args.outputFile
+    local = args.local
+
+    return (cooccurrenceFile, logFile, outputFile, local)
+
+
+
+if __name__ == '__main__':
```

Comment:
PEP8 again; exactly two blank lines between top level statements

```suggestion
    return (cooccurrenceFile, logFile, outputFile, local)


if __name__ == '__main__':
```